File: src/SimpleApp/Simple.AdminApplication/TenantMng/Entities/TenantPackage.cs
namespace Simple.AdminApplication.TenantMng.Entities
{
    /// <summary>
    /// 租户套餐表
    /// </summary>
    public class TenantPackage : DefaultNoTenantEntity
    {
        /// <summary>
        /// 套餐名称
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// 描述
        /// </summary>
        public string Description { get; set; }

        /// <summary>
        /// 用户数量
        /// </summary>
        public int UserCount { get; set; }
    }
}

File: vue-vben-admin/apps/web-naive/src/api/entity/TenantPackage.ts
export interface TenantPackage {
  name: string  | null | undefined ;    //  套餐名称
  description: string  | null | undefined ;    //  描述
  userCount: number  | null | undefined ;    //  用户数量
  id: string  | null | undefined ;    //  主键
  createdId: string  | null | undefined ;    //  创建人GUID
  createdTime: string  | null | undefined ;    //  创建时间
  creator: string  | null | undefined ;    //  创建人
  updatedTime: string  | null | undefined ;    //  更新时间
  updatedId: string  | null | undefined ;    //  更新人GUID
  updator: string  | null | undefined ;    //  更新人
}
